src/Sql/Sql/ManagedInstance/Cmdlet/NewAzureSqlManagedInstance.cs

```diff
@@ -139,6 +139,15 @@ public class NewAzureSqlManagedInstance : ManagedInstanceCmdletBase
         [PSArgumentCompleter(Constants.ComputeGenerationGen4, Constants.ComputeGenerationGen5)]
         public string ComputeGeneration { get; set; }
 
+        /// <summary>
+        /// Gets or sets the instance collation
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The collation of the Azure SQL Managed Instance to use.")]
+        [ValidateNotNullOrEmpty]
+        [PSArgumentCompleter("SQL_Latin1_General_CP1_CI_AS", "Latin1_General_100_CS_AS_SC")]
+        public string Collation { get; set; }
```

Comment:
can't you define an enum of possible values?

Reply:
Done.